deploy/helm/baserow/README.md

```diff
@@ -92,6 +92,34 @@ baserow-backend-wsgi:
       onDemandAsk: "http://my-baserow-baserow-backend-wsgi/api/builder/domains/ask-public-domain-exists/"
 ```
 
+## AI and Embeddings Configuration
+
+Baserow supports multiple AI providers for generative AI features and the AI assistant. The embeddings service powers semantic search for the AI assistant's documentation lookup feature.
+
+### Enable Embeddings Service
+
+The AI assistant uses the embeddings service and requires an LLM model to be configured. The embeddings service is automatically configured when enabled.
+
+#### Basic Configuration
+
+```yaml
+baserow-embeddings:
+  enabled: true
+  assistantLLMModel: "groq/openai/gpt-oss-120b"
+
+backendSecrets:
+  # Add the appropriate API key for your chosen model provider
+  GROQ_API_KEY: "gsk_..."
+```
+
+The assistantLLMModel option sets the LLM model to use for the AI assistant it follows the [LiteLLM provider format](https://docs.litellm.ai/docs/providers)
```

Comment:
We recently got rid of the DSPy dependency, and therefore the also the LiteLLM dependency. This is because LiteLLM needs 200MB of memory per worker, and that leads to a significant increase.

Please take a look at the docs here on what is accepted https://github.com/baserow/baserow/blob/develop/docs/installation/ai-assistant.md.